src/fer/fer.py

```diff
@@ -86,17 +87,17 @@ def __init__(
         self.tfserving = tfserving
 
         if cascade_file is None:
-            cascade_file = cv2.data.haarcascades + 'haarcascade_frontalface_default.xml'
+            cascade_file = cv2.data.haarcascades + "haarcascade_frontalface_default.xml"
 
         if mtcnn:
             try:
-                from mtcnn.mtcnn import MTCNN
+                from facenet_pytorch import MTCNN
             except ImportError:
                 raise Exception(
-                    "MTCNN not installed, install it with pip install mtcnn"
+                    "MTCNN not installed, install it with pip install facenet-pytorch and from facenet_pytorch import MTCNN"
                 )
             self.__face_detector = "mtcnn"
-            self._mtcnn = MTCNN()
+            self._mtcnn = MTCNN(keep_all=True, device="cuda:0")
```

Comment:
Is specifying cuda necessary here? Brings error on running py.test on my Mac (CPU only):
` AssertionError: Torch not compiled with CUDA enabled`